site/components/CopyString.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <button class="text-left tooltip transition-colors" :data-tip="tooltipMessage" type="button" @click="copyUserIdToClipboard">{{ $props.content }}</button>
+</template>
+
+<script setup lang="ts">
+
+const initialTooltip = "copy to clipboard";
+
+const tooltipMessage = ref(initialTooltip);
+const highlightColor = "text-emerald-500";
+
+const props = defineProps<{
+    content: String | undefined,
```

Comment:
There is a shorthand for this type (and allows for the value to not exist in the obj).
```suggestion
    content?: string,
```